src/cursor.js

```diff
@@ -92,8 +92,21 @@ export default class Cursor {
     // Without setting focus() Firefox is not happy (seems setting a selection is not enough.
     // Probably because Firefox can handle multiple selections).
     if (this.win.document.activeElement !== this.host) {
+      const previousWindowTop = $(window).scrollTop()
+
       $(this.host).focus()
+
+      const hostBottom = $(this.host).position().top + $(this.host).outerHeight()
```

Comment:
I like the idea but I think that might a bit too much for a low level library as editable. Plus I'm not sure if the pasted text is large if the cursor is still visible in all cases.

Reply:
And here's an example that doesn't use jquery: https://github.com/upfrontIO/editable.js/commit/83eb19c7e4d3fb6c388d853c9ff0a1b5d5b50e7e

We try to move away from jQuery in editable. So I try to do new code without.

Reply:
And we have existing code that reads the scroll position in this file. So we can reuse that logic.

Reply:
👍